assets/js/theme/common/carousel/utils/arrowAriaLabling.js

```diff
@@ -0,0 +1,26 @@
+const NUMBER = '[NUMBER]';
+const integerRegExp = /[0-9]+/;
+const lastIntegerRegExp = /(\d+)(?!.*\d)/;
+
+export default ($prevArrow, $nextArrow, realCurrentSlide, realSlideCount) => {
+    if (realSlideCount < 2) return;
+    if ($prevArrow.length === 0 || $nextArrow.length === 0) return;
+
+    const arrowAriaLabelBaseText = $prevArrow.attr('aria-label');
+
+    const isInit = arrowAriaLabelBaseText.includes(NUMBER);
+    const valueToReplace = isInit ? NUMBER : integerRegExp;
+    const currentSlideNumber = realCurrentSlide + 1;
+
+    const rightGoToNumber = realCurrentSlide === realSlideCount - 1 ? 1 : currentSlideNumber + 1;
```

Comment:
won't be rightNumberToMove easier to read?

Reply:
nextSlideNumber?

Reply:
yes, nextSlideNumber is better to read)